lib/load.php

```diff
@@ -61,6 +61,9 @@ function gutenberg_is_experiment_enabled( $name ) {
 	if ( ! class_exists( 'WP_REST_Templates_Controller' ) ) {
 		require_once __DIR__ . '/full-site-editing/class-wp-rest-templates-controller.php';
 	}
+	if ( ! class_exists( 'WP_REST_Block_Editor_Global_Styles_Settings_Controller' ) ) {
+		require dirname( __FILE__ ) . '/class-wp-rest-block-editor-global-styles-settings-controller.php';
```

Comment:
Lets use `require_once` here so that the file can't be loaded multiple times for some reason.

Reply:
Thanks for the suggestion! I'll change it 👍